src/lazy/any_encoding.rs

```diff
@@ -204,8 +205,15 @@ impl<'data> LazyRawValue<'data, AnyEncoding> for LazyRawAnyValue<'data> {
         }
     }
 
-    fn annotations(&self) -> <AnyEncoding as LazyDecoder<'data>>::AnnotationsIterator {
-        todo!()
```

Comment:
🗺️ As with the other `LazyRawAnyReader` methods, we branch on which encoding we're reading and delegate the method call to the appropriate reader.